src/inc/msquic.h

```diff
@@ -720,6 +720,14 @@ void
 #ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
 #define QUIC_PARAM_CONFIGURATION_VERSION_SETTINGS       0x03000002  // QUIC_VERSION_SETTINGS
 #endif
+#ifdef WIN32 // Schannel-specific Configuration parameters
```

Comment:
Since this is **part** of the configuration space, I think we should not wrap it in `#ifdef WIN32` but instead purely comment the dependency. This means spinquic can "try" it out on other platforms too, to verify it doesn't crash.